VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -540,8 +540,10 @@ def get_wala_version_from_command(self):
             while(process_wait_time > 0 and p.poll() is None):
                 time.sleep(1)
                 process_wait_time -= 1
-            out = p.stdout.read()
+            out = p.stdout.read().decode()
```

Comment:
will this decode not cause an issue with plain strings?
i.e. already existing VMs where byte array is not coming

Reply:
No. It doesn't throw an error for normal string. I tested it.